groovy/prePostInit/oreDict.groovy

```diff
@@ -93,6 +93,8 @@ ore('oreBauxite').add(item('susy:resource_block:0'))
 ore('oreSulfur').add(item('susy:resource_block:14'))
 ore('oreCoalDeposit').add(item('susy:resource_block:15'))
 ore('oreNativeCopperDeposit').add(item('susy:resource_block_1:0'))
+ore('oreAnthraciteDeposit').add(item('susy:resource_block_1:1'))
+ore('oreLigniteDeposit').add(item('susy:resource_block_1:2'))
```

Comment:
to let scanners consider this block as an gt ore, you need to register an material for it, you can refer to the [coal deposit commit](https://github.com/SymmetricDevs/Supersymmetry/commit/552ec62a24e5f68ff318196eecc507caf0ee1898)

Reply:
done